packages/pinia/src/storeToRefs.ts

```diff
@@ -120,6 +120,6 @@ export function storeToRefs<SS extends StoreGeneric>(
       }
     }
 
-    return refs
+    return { ...store.actions, ...refs }
```

Comment:
Thanks but this is intended

Reply:
I know I know. This is just an example for the debate @posva . This is what I mentioned it would be an idea to change.